library/lib_location_constants.ks

```diff
@@ -5,7 +5,7 @@ global location_constants is lex().
 //runways should be added in this pattern "name_runway_##_start" where name is the name of the runway and ## is the runway number
 //  If not added with this pattern then the automatic alias creation that add an end position based on the start of the same name but opposite number will fail
 
-if bodyexists("kerbin") and bodyexists("mun") and bodyexists("minmus") {//check for if the craft is in the stock solar system
+if bodyexists("Sun") and body("Sun"):radius = 261_600_000 {//check for if the craft is in the stock solar system
```

Comment:
I think it's probably wise to include `bodyexists("kerbin")` as well before using code that references `Kerbin`.

Side note: I somehow missed or forgot that you could add in underscore separators in scalar literals. Definitely improves readability!

Reply:
Good point, done